assembl/static2/js/app/components/common/participateButton.jsx

```diff
@@ -8,14 +8,15 @@ import { getIsDebateStarted } from '../../utils/timeline';
 type Props = {
   displayPhase: Function,
   timeline: Timeline,
-  btnClass: string
+  btnClass: string,
+  btnLabel: string | null
```

Comment:
@ayyazdaniaryan `btnLabel?: string`. It means optional value.

Reply:
otherwise if it is not optional but can be null you can write ```btnLabel: ?string``` instead of ```| null```